Snakefile

```diff
@@ -34,370 +34,17 @@ rule all:
-    resources:
-        # Memory use scales primarily with the size of the metadata file.
-        mem_mb=20000
-    shell:
-        """
-        ./bin/flag-metadata {input.metadata} > {output.metadata}
-        """
-
-rule notify_gisaid:
-    input:
-        flagged_annotations = rules.transform_gisaid_data.output.flagged_annotations,
-        # metadata = "data/gisaid/metadata.tsv",
-        additional_info = "data/gisaid/additional_info.tsv",
-        flagged_metadata = "data/gisaid/flagged_metadata.txt"
-    params:
-        s3_bucket = config["s3_src"]
-    output:
-        touch("data/gisaid/notify.done")
-    run:
-        shell("./bin/notify-slack --upload flagged-annotations < {input.flagged_annotations}")
-        # notify-on-metadata-change disabled as csv-diff runs out of memory
-        # shell("./bin/notify-on-metadata-change {input.metadata} {params.s3_bucket}/metadata.tsv.gz gisaid_epi_isl")
-        shell("./bin/notify-on-additional-info-change {input.additional_info} {params.s3_bucket}/additional_info.tsv.gz")
-        shell("./bin/notify-on-flagged-metadata-change {input.flagged_metadata}  {params.s3_bucket}/flagged_metadata.txt.gz")
-
-rule notify_genbank:
-    input:
-        flagged_annotations = rules.transform_genbank_data.output.flagged_annotations,
-        duplicate_biosample = "data/genbank/duplicate_biosample.txt"
-    params:
-        s3_bucket = config["s3_src"]
-    output:
-        touch("data/genbank/notify.done")
-    run:
-        shell("./bin/notify-slack --upload flagged-annotations < {input.flagged_annotations}")
-        # TODO - which rule produces data/genbank/problem_data.tsv? (was not explicit in `ingest-genbank` bash script)
-        shell("./bin/notify-on-problem-data data/genbank/problem_data.tsv")
-        shell("./bin/notify-on-duplicate-biosample-change {input.duplicate_biosample} {params.s3_bucket}/duplicate_biosample.txt.gz")
-
-
-def compute_files_to_upload(wildcards):
-    files_to_upload = {
-                        "metadata.tsv.gz":              f"data/{database}/metadata.tsv",
-                        "sequences.fasta.xz":           f"data/{database}/sequences.fasta"}
-    if database=="genbank":
-        files_to_upload["biosample.tsv.gz"] =           f"data/{database}/biosample.tsv"
-        files_to_upload["duplicate_biosample.txt.gz"] = f"data/{database}/duplicate_biosample.txt"
-    elif database=="gisaid":
-        files_to_upload["additional_info.tsv.gz"] =     f"data/{database}/additional_info.tsv"
-        files_to_upload["flagged_metadata.txt.gz"] =    f"data/{database}/flagged_metadata.txt"
-
-    nextclade_sequences_path = checkpoints.get_sequences_without_nextclade_annotations.get().output.fasta
-    if os.path.getsize(nextclade_sequences_path) > 0:
-        files_to_upload["nextclade.tsv.gz"] =           f"data/{database}/nextclade.tsv"
-        files_to_upload["aligned.fasta.xz"] =           f"data/{database}/aligned.fasta"
```

Comment:
_non-blocking, minor_

I noticed that git's moved line detection flagged these two lines as not being moved, i.e. the original lines were deleted here but different lines added over there instead of the same lines added. When I looked into why, it was just an internal whitespace change, presumably accidental. Here's [a fixup patch](https://github.com/nextstrain/ncov-ingest/files/9203539/0001-fixup-Split-Snakefile-into-workflow-modules.txt) for that, which you can apply if you want with:

```
curl -fsSL https://github.com/nextstrain/ncov-ingest/files/9203539/0001-fixup-Split-Snakefile-into-workflow-modules.txt \
  | git am
```

and then actually fixup the original commit with `git rebase -i`.

Reply:
Ah, thank you for the patch!